### DISCUSSION TITLE
SS. Angeli Custodes: First Sunday in September

### DISCUSSION THREAD

**Discussion by Augustinus-Altovadensis** [DivinumOfficium/divinum-officium#4014] [Q&A] [answered]
Laudétur Jesus Christus!

I have a question: in the Cistercian calendar, specifically in Bohemia and Lusatia, there are two feasts of Guardian Angels. The first one is on the first Sunday in September with the rank of Serm. maj. (the highest one), another is fixed on October 2. 

Now my question: is there any (simple) way to enter the first Sunday in September? Or should I use something similar to the Vigil of St. Mathias (there used to be more cases like this in the code, if I remember correctly)? I mean I know about the `Tempora/091-0.txt` file, but it doesn't seem like the correct way...

Thanks a lot, God bless!

**Comment by FAJ-Munich** [comment, marked as answer]
It depends on what makes it easier to handle the occurrence and concurrence against the remainder of the Sanctorale. Since Cistercian still transfers everything from the rank of XII lect et Missa and above, using 091-0.txt probably makes things more complicated then necessary. 
I would put the feast in a separate file in the Sanctorale, e.g.  09-DP.txt (for Dominica Prima) and then assign the feast and the corresponding transferrals across the transfer files a.txt through g.txt. 
You have to make the feast known to the ordo as well by assiningn it in the Kalendarium as well. 
A line like 09-00=09-DP=... should do.

**Reply by Augustinus-Altovadensis** [reply]
Thank you, this looks clean and simple enough. Now I'll just have to find the corresponding dates.

BTW, as those Guardian Angels are Serm. maj., it doesn't go anywhere, and the only "feast" it can collide with is the Sunday, on which we _know_ it'll fall and supersede... 
Also, the brothers from Vyšší Brod decided to take over one thing from 1957, namely that Sunday supersedes MM. min., so the transfers of xij. Lect. et M. and higher change, and this concern also the solemn Octaves.
